ruby-challenges.rb

```diff
@@ -19,10 +25,49 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum_array array
+  array.inject(0) { |sum, num| sum += num }
```

Comment:
Nice!  There is also a Ruby built-in method called sum that is a little simpler and is faster for the computer to process as well.